### ISSUE TITLE
Move favorites selection to context menu

### ISSUE THREAD

**Issue by Gurkenglas** [falzonv/discreet-launcher#118]
The favorites selector is a worse reimplementation of the entire launcher. Instead, after [`Open ...`, `View store page`, `Access its settings`], put `Toggle favorite`.

**Comment by falzonv** [comment]
Hello,

Thank you for the suggestion, I will have a look.

However the alternative you propose doesn't allow to manage the order of favorites (if you don't like that part either).
I know several launchers do this with a long-click and icon drag-and-drop, but in this case the long-click is already assigned to other functions and the icon drag-and-drop stuff would add a lot of unnecessary complexity anyway.

*Remark: not sure what you mean by "reimplementation" since I created this launcher from scratch.*

Best regards.

**Comment by Gurkenglas** [comment]
I didn't mean to insult the main panel with the app icons, I meant the `SELECT FAVORITES` menu where there is a checkbox for every app.

The reordering menu, I suppose, could be integrated into the main panel via a context menu entry that initiates a relocation of the selected button, but I admit this additional step is less obviously an improvement.

**Comment by falzonv** [comment]
Hello @Gurkenglas,

This has been a long time but long press options to add/remove favorites have been added in new release v5.2.0!
I will leave this ticket open until it is available on F-Droid (usually takes 4 to 8 days).

For the favorites order, I tried a couple of alternatives but will finally stay with the current implementation (it allows to avoid a lot of code complexity and maintenance troubles).

Best regards.
